mad-dog/src/tests/test_integration.js

```diff
@@ -176,6 +186,84 @@ module.exports = coreIntegration = async ({
         emit(Event.RESPONSE, res)
         break
       }
+      case OPERATION_TYPE.CREATE_PLAYLIST: {
+        try {
+          // create playlist
+          const randomPlaylistName = genRandomString(8)
+          const playlist = await executeOne(walletIndex, l =>
+            createPlaylist(l, userId, randomPlaylistName, false, false, [])
+          )
+          await executeOne(walletIndex, l => l.waitForLatestBlock())
+
+          // verify playlist
+          const verifiedPlaylist = await executeOne(walletIndex, l =>
+            getPlaylists(l, 100, 0, [playlist.playlistId], userId)
+          )
+          if (verifiedPlaylist[0].playlist_id !== playlist.playlistId) {
+            throw new Error(`Error verifying playlist [${playlist.playlistId}]`)
+          }
+
+          res = new CreatePlaylistResponse(walletIndex, playlist.playlistId, userId)
+        } catch (e) {
+          logger.error(`Caught error [${e.message}] creating playlist.\n${e.stack}`)
+          res = new CreatePlaylistResponse(walletIndex, null, userId)
+          throw new Error(e)
+        }
+        emit(Event.RESPONSE, res)
+        break
+      }
+      case OPERATION_TYPE.ADD_PLAYLIST_TRACK: {
+        if (uploadedTracks.length === 0 || !createdPlaylists[userId]) {
+          res = new AddPlaylistTrackResponse(
+            walletIndex,
+            null,
+            false,
+            new Error('Adding a track to a playlist requires a track to be uploaded.')
+          )
+        } else {
+          const trackId = uploadedTracks[_.random(uploadedTracks.length - 1)].trackId
+          try {
+            // add track to playlist
+            const playlistId = createdPlaylists[userId][createdPlaylists[userId].length - 1]
+            await executeOne(walletIndex, l =>
+              addPlaylistTrack(l, playlistId, trackId)
```

Comment:
Might be worth adding a log specifying which track is added to which playlist for which user